### PR TITLE
namespace change added for 0.5.3

### PR DESCRIPTION
**Pull request by mahebbar**
Signed-off-by: Madhuri Hebbar <madhuri.hebbar@cloudbyte.com>

### CODE REVIEW

File: docs/namespace.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                fieldRef:
+                  fieldPath: metadata.namespace
+
+### Namespace in Application Files
+
+You can also use the following commands to make changes in the application files. In the following example you are creating a percona namespace.
+
+```
+kubectl create ns percona
+kubectl apply -f openebs-operator.yaml -n percona
```
Comment (yudaykiran) [inline docs/namespace.md]: kubectl apply -f percona.yaml -n percona

File: docs/namespace.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+---
+id: namespace
+title: Customizing Namespace
+sidebar_label: Customizing Namespace
+---
+
+OpenEBS is usually deployed in the *default* namespace. You can deploy OpenEBS in another namespace  other than the *default* namespace.  To customize namespace to *openebs* instead of *default*,  you must change all the namespaces in the *openebs-operator.yaml* file to *openebs*.
```
Comment (yudaykiran) [inline docs/namespace.md]: you must replace all occurrences of the default namespace with namespace openebs.

File: docs/namespace.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+1. Create a namespace as in the following example i.e., modify *name:* to *openebs*.
+
+   ```
+   apiVersion: v1
+   kind: Namespace
+   metadata:
+     name: openebs
+   ```
+
+2. Change all occurrences of *namespace: default* to *namespace: openebs* in the *openebs-operator.yaml* file.
```
Comment (yudaykiran) [inline docs/namespace.md]: Looks redundant, can it be removed here or from the earlier point?

File: docs/namespace.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+kubectl create ns percona
+kubectl apply -f openebs-operator.yaml -n percona 
+```
+
+You can view the changes by giving the `kubectl get pods --all-namespaces` command which will display output similar to following. You can see that the default namespace is replaced with percona.
+
+```
+vagrant@minikube-dev:~$ kubectl get pods --all-namespaces
+NAMESPACE     NAME                                                             READY     STATUS   RESTARTS   AGE
+percona       maya-apiserver-7cd7478c74-xrxnl                                  1/1       Running   4          8d
```
Comment (yudaykiran) [inline docs/namespace.md]: maya-apiserver and openebs-provisioner under the openebs namespace.

File: docs/namespace.md (hunk 1 above)
Comment (mahebbar) [reply docs/namespace.md]: Done

File: docs/namespace.md (hunk 3 above)
Comment (mahebbar) [reply docs/namespace.md]: Done

**Pull request merged**